src/Core/Localization/Currency.php

```diff
@@ -89,7 +89,7 @@ class Currency implements CurrencyInterface
     /**
      * Number of decimal digits to use with this currency.
      *
-     * @var int
+     * @var int|null
```

Comment:
```suggestion
     * @var int
```
Now this one is true